pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -345,8 +345,30 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(cniTimeoutCtx, plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
-	return fmt.Sprintf("%s_%s/%s", namespace, name, id.ID)
+func getIpsFromResult(res *cnicurrent.Result, ifaceName string) []net.IP {
```

Comment:
Any reason this method isn't exported ? This could be useful for kubenet too: https://github.com/kubernetes/kubernetes/pull/82844/files#r329884258

Reply:
I can change this in my PR no need to do this in here. Just a question if there are any reasons not to use this method in `kubenet`.

Reply:
@johscheuer, @dcbw I think if function is used by libraries which are on the same level (layer), like cni/kubenet we need to move that function to common library. Is k8s.io/kubernetes/pkg/kubelet/dockershim/network a common library for this case?

Reply:
IMHO I would add here a second method: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/network/plugins.go#L260 e.g.`GetPodIPsFromCNIResult`

Reply:
Ok, I can split this commit and add this method there, but tomorrow.